modules/claims_api/app/sidekiq/claims_api/special_issue_updater.rb

```diff
@@ -150,7 +150,11 @@ def existing_contentions(claim, contention_id, special_issues)
     def existing_special_issues(contention, special_issues = [])
       contention[:special_issues] = [] if contention[:special_issues].blank?
 
-      unique_special_issues = (special_issues + contention[:special_issues].pluck(:spis_tc)).uniq
+      # Let's just always know it is an array
+      contentions =
+        contention[:special_issues].is_a?(Hash) ? [contention[:special_issues]] : contention[:special_issues]
```

Comment:
Could we just use [Array.wrap](https://api.rubyonrails.org/v3.1/classes/Array.html#:~:text=Class%20Public%20methods-,wrap,-(object))? No need for a comment.

```suggestion
      contentions = Array.wrap(contention[:special_issues])
```

Reply:
Yep works perfectly [58de957](https://github.com/department-of-veterans-affairs/vets-api/pull/20324/commits/58de9577308181e2ea635baf8d9684e3e34fd072)